build.sbt

```diff
@@ -88,15 +89,18 @@ resolvers ++= Seq(
     Resolver.bintrayRepo("guardian", "ophan")
 )
 
-import com.typesafe.sbt.packager.archetypes.ServerLoader.Systemd
-serverLoading in Debian := Systemd
+import com.typesafe.sbt.packager.archetypes.systemloader.ServerLoader.Systemd
+enablePlugins(SystemdPlugin)
+serverLoading in Debian := Some(Systemd)
 debianPackageDependencies := Seq("openjdk-8-jre-headless")
 maintainer := "Membership Dev <membership.dev@theguardian.com>"
 packageSummary := "Subscription Frontend"
 packageDescription := """Subscription Frontend"""
 
 riffRaffPackageType := (packageBin in Debian).value
 
+routesGenerator := StaticRoutesGenerator
```

Comment:
It looks like you've converted all the controllers to classes so you don't need this anymore and if you get rid of it you can get rid of the @ in front of the controller name in all the routes.

Reply:
That change causes a rather unfriendly java.lang.RuntimeException: There is no started application